config.json

```diff
@@ -597,7 +656,16 @@
         "slug": "zipper",
         "name": "Zipper",
         "uuid": "bc315734-051c-4735-9a8a-3aacb094d2ca",
-        "prerequisites": [],
+        "prerequisites": [
+          "recursion",
+          "structs",
+          "protocols",
```

Comment:
I don't think protocols are necessary to solve this exercise, but the boilerplate code includes a protocol implementation (https://github.com/exercism/elixir/blob/main/exercises/practice/zipper/lib/bin_tree.ex#L15-L33) so I added it here to avoid students being confused by the boilerplate.

Reply:
I think Zipper used to appear after Clock, so they were introduced to protocols by that point. I don't think students that I have encountered have been confused by this at this point with Clock having been removed from the core pathway.